as/plugins/org.jboss.ide.eclipse.as.ui/jbossui/org/jboss/ide/eclipse/as/ui/JBossServerUIPlugin.java

```diff
@@ -124,10 +124,6 @@ public static JBossServerUIPlugin getDefault() {
 	public ResourceBundle getResourceBundle() {
 		return resourceBundle;
 	}
-
-	public void earlyStartup() {
-		JBossServerCorePlugin.getDefault();
-	}
```

Comment:
It is bad practice to have a plug-in activator and early workspace start up implemented in one class, so I split it and added class attribute which pointing to new ISturtup implementation.  
earlyStartup method is empty, because it will trigger bundle activation on class load and everything else are going to work the same way as before, but no anti patterns and no error markers in manifest.